homeworks/serhii.noskov_SerhiiNSN/GamesProject/games.html

```diff
@@ -0,0 +1,346 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link
+            href="https://fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,regular,italic,500,500italic,700,700italic,900,900italic"
+            rel="stylesheet"
+        >
+        <link rel="stylesheet" href="css/style.css">
+        <title>GamesProject</title>
+    </head>
+
+    <body>
+        <header class="header">
+            <div class="header__container">
+                <a class="header__logo" href="#">
+                    <img src="img/logo.svg" alt="logo">
+                </a>
+                <nav class="nav">
+                    <ul class="header__menu">
+                        <li class="menu__list">
+                            <a href="games.html" class="menu__link">Games</a>
+                        </li>
+                        <li class="menu__list">
+                            <a href="index.html" class="menu__link">About</a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <main class="main">
+            <section class="games">
+                <div class="games__container">
```

Comment:
If you have several simular elements - it's a list. So, it is a list of cards. Please update the structure

Reply:
I have added the list, waiting for verification)

Reply:
It seems there might be a misunderstanding. What I meant is that you have multiple similar cards; it's essentially a list of cards. 

For ex. :
```
<ul class="games__list">      //   (games__body - in your code)
   <li class="games__card">
       All card here
       ....
   </li>
</ul>
```
   
 The card content might be:
` <p><span class="bold"> Genre </span> some text </p>`

Reply:
I hope I have understood you correctly now)